### ISSUE TITLE
[BUG] Text capitalization on new sentences not working on iOS

### ISSUE THREAD

**Issue by steveneal** [the-airbender/quill_html_editor#36]
**Describe the bug**

When pressing return in the editor, the keyboard should return to uppercase mode as a new sentence is about to start. This works nicely on Android, but not on iOS. 

**To Reproduce**
1. Run editor on iPhone.
2. Type some content followed by a new line / return.

**Expected behavior**
The keyboard should switch to caps as we're starting a new sentence.

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone SE and 14]
 - OS: [e.g. iOS 15.x-16.x]

**Flutter Doctor**
`
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.3.9, on macOS 13.0.1 22A400 darwin-arm, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2022.1)
[✓] VS Code (version 1.76.0)
[✓] Connected device (4 available)
    ! Device emulator-5554 is offline.
[✓] HTTP Host Availability

• No issues found!
`

**Comment by gerken-tss** [comment]
@steveneal 
To be even more precise, shouldn't this be a setting? Like all normal `TextField`s in Flutter have this [TextCapitalization](https://api.flutter.dev/flutter/services/TextCapitalization.html) enum or property.
What you describe is `TextCapitalization.senctences`.

**Comment by the-airbender** [comment]
@steveneal and @gerken-tss ,

I believe this is a default behaviour of the Quill js editor on iOS devices. 

However, we have a workaround for this. Will add this fix in coming release, incase if it works.

Thanks!

**Comment by steveneal** [comment]
@gerken-tss yes this would be ideal; but for my use case, I need capitalisation on new sentences.

@the-airbender thanks, your work in this plugin is very much appreaciated.
